x-pack/plugins/security_solution/public/management/components/endpoint_responder/endpoint_response_actions_console_commands.ts

```diff
@@ -56,6 +61,36 @@ export const getEndpointResponseActionsConsoleCommands = (
         },
       },
     },
+    {
+      name: 'kill-process',
+      about: i18n.translate('xpack.securitySolution.endpointConsoleCommands.killProcess.about', {
+        defaultMessage: 'Kill a running process',
+      }),
+      RenderComponent: KillProcessActionResult,
+      meta: {
+        endpointId: endpointAgentId,
+      },
+      exampleUsage: 'kill-process --pid 123',
+      exampleInstruction: 'Enter a pid or entity id to execute',
+      args: {
+        comment: {
+          required: false,
+          allowMultiples: false,
+          about: i18n.translate(
+            'xpack.securitySolution.endpointConsoleCommands.release.arg.comment',
+            { defaultMessage: 'A comment to go along with the action' }
+          ),
+        },
+        pid: {
+          required: true,
+          allowMultiples: false,
+          about: i18n.translate('xpack.securitySolution.endpointConsoleCommands.pid.arg.comment', {
+            defaultMessage:
+              'A PID representng the process to kill.  You can enter a pid or entity id.',
```

Comment:
So wait, there is now only one flag? They both are `kill --pid 34` OR `kill --pid XYZ` for entity?

If:
- that is the case, that the user no longer has to specify _whether_ it is a PID or entity (since we are trying to parse and determine) AND
- it is a required argument (i.e. `kill` is not valid without the PID)

_then_

the interface should be different. Console (and console-like) commands have very common patterns. If a command takes a single required argument, the command is the form:

`kill 34`

No flags. `--pid` is just extra unnecessary typing. dashes and flags normally represent **optional** arguments, _unless_ you have something with many positions and you want to use the flag names to disambiguate. If we are no longer disambiguating, this should not be a flag any more.

And yes, I know we haven't set up the parser to do that, I'm not sure it's been warranted until this change.

comment remains an example of a perfect optional flag here.

So the command **should be**:

`kill [--comment user comment] <pid-or-entity>`

Reply:
@pzl 

We chatted offline and we determined that we should still enforce that the user chooses between `--pid` and `--entityId`.  I've reverted the logic here, so this issue is no longer a concern in our supported commands (for now).